.changeset/cyan-beans-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@biomejs/biome": patch
+---
+
+Fixed properties of object type annotation in function parameters are expanded, which is inconsistent with Prettier.
```

Comment:
nit:
```suggestion
Fixed properties of object type annotation in function parameters are expanded, which was inconsistent with Prettier.
```

Might be good to also link the fixed issue too.

Reply:
Fixed in [37bb3d5](https://github.com/biomejs/biome/pull/5174/commits/37bb3d5059fc36be753034b0ae2f088270c9c7f2), thank you for the suggestion